packages/form-generator/src/local_codegen_graphql_form_generator.ts

```diff
@@ -33,7 +33,7 @@ export type RenderOptions = {
 
 export type ResultBuilder = (
   fileMap: Record<string, string>
-) => GraphqlGenerationResult;
+) => CodegenGraphqlFormGeneratorResult;
```

Comment:
Don't depend on a specific implementation of the generic interface.

Reply:
Yeah, I can revert this since I can get the writeFile content from mock call